Assets/MixedRealityToolkit/InputModule/Scripts/InputManager.cs

```diff
@@ -245,8 +247,20 @@ private void Start()
 
             Debug.Assert(!eventData.used);
 
-            // Use focused object when OverrideFocusedObject is null.
-            GameObject focusedObject = (OverrideFocusedObject == null) ? FocusManager.Instance.TryGetFocusedObject(eventData) : OverrideFocusedObject;
+            
+            GameObject focusedObject;
+
+            if (OverrideFocusedObject != null)
+            {
+                // Use override focused object if not null.
+                focusedObject = OverrideFocusedObject;
+                eventData.selectedObject = OverrideFocusedObject;
```

Comment:
we don't generally use/set `eventData.selectedObject` unless it's for uGUI related things. generally this is null.

We should revert these lines.